src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -1961,7 +1961,6 @@ class Browser extends DashboardView {
             setRelation={this.setRelation}
             onAddColumn={this.showAddColumn}
             onAddRow={this.addRow}
-            onAbortAddRow={this.abortAddRow}
```

Comment:
Why has this been removed?

Reply:
My IDE highlighted that line as a duplicated binding, and indeed it was a [duplicate](https://github.com/parse-community/parse-dashboard/pull/2581/files#diff-a36fee044b77084dd772401ef5f1d8281428f30c5f099d6e7f9661498a7be3aaR1934).

Reply:
Good catch!